### PR TITLE
Add Learn/Blogs: MLflow 101 — Local Experiment Tracking for Beginners

### LINKED ISSUE #149: [Resource] MLflow 101: Local Experiment Tracking for Beginners
Label requested: Resource

I’d like to add a short, beginner-friendly guide that shows how to track ML experiments locally with MLflow. The post walks through logging params/metrics/artifacts from a small scikit-learn script and comparing runs in the MLflow UI. It’s focused on reproducibility for class projects, labs, and hackathons with zero cloud setup.

**Audience & prerequisites.** Early-stage ML students or anyone new to experiment tracking. Assumes basic Python/sklearn and light Git/GitHub familiarity. I will link to relevant Nexus pages (Intro to sklearn workshop, GitHub Desktop guide, and the reproducibility overview) for background.

### PR DESCRIPTION
**Pull request by Dev-PSR**
Beginner-friendly guide to local experiment tracking with MLflow. Shows how to log params/metrics/artifacts from a small scikit-learn script and compare runs in the MLflow UI. Designed for class projects, labs, and hackathons; runs fully on a laptop with free tools.

Closes #149
Label requested: Resource

### CODE REVIEW

File: Learn/Blogs/mlflow-101.qmd [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: "MLflow 101: Local Experiment Tracking for Beginners"
+author: 
+  - name: Parith Reddy  
+    email: psreddy@wisc.edu
+
+date: 2025-08-13  
+date-format: long  
+
```
Comment (qualiaMachine) [inline Learn/Blogs/mlflow-101.qmd:9]: Missing "image" field. Please upload the logo for ML flow to our images folder. 
`image: "../../../images/vincent-van-zalinge-mDohQISBnCk-unsplash.jpg" # In your local copy of the Nexus repository, add a representative image for this resource to the images folder. Both PNGs and JPGs are acceptable formats. Adjust the filename here from "vincent-van-zalinge-mDohQISBnCk-unsplash.jpg" to the name of your image.`

File: Learn/Blogs/mlflow-101.qmd [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+- Version Control with GitHub Desktop (Nexus): https://uw-madison-datascience.github.io/ML-X-Nexus/Learn/Guides/Github-desktop.html
+
+Write your blog content here. Discuss the following (if applicable):
+
+- **Motivation**: Many students keep “best results” in scattered notebook cells or screenshots; MLflow makes experiments comparable and reproducible.
+- **Problem/Challenge**: Track hyperparameters, metrics, and artifacts so you can iterate with evidence instead of memory.
+- **Methods/Techniques**: Use MLflow’s Python API to log params/metrics and artifacts from a scikit-learn `LogisticRegression` baseline; view and compare runs in the local MLflow UI.
+- **Results**: Side-by-side comparisons of AUC/accuracy across different `C` values; a tiny, reusable template you can drop into any future project.
+
+## Code Snippets (Optional)
```
Comment (qualiaMachine) [inline Learn/Blogs/mlflow-101.qmd:31]: Please remove any instructions that aren't content

File: Learn/Blogs/mlflow-101.qmd [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+If you're sharing code directly, include short snippets or explanations of the key components below. You can also link to a repo later.
```
Comment (qualiaMachine) [inline Learn/Blogs/mlflow-101.qmd:32]: Please remove any instructions that aren't content

File: Learn/Blogs/mlflow-101.qmd [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+```python
```
Comment (qualiaMachine) [inline Learn/Blogs/mlflow-101.qmd:34]: If including code, please talk about the code before it is shown. What are you trying to show here? Should be clear without looking at code.

File: Learn/Blogs/mlflow-101.qmd [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+categories: 
+  - Blogs
+  - Reproducibility
+  - Sklearn
+  - MLflow
+  - Code-along
+
+---
+This blog is a beginner-friendly walkthrough of local experiment tracking with [MLflow Tracking](https://mlflow.org/docs/latest/tracking.html). You’ll learn how to log parameters, metrics, and artifacts from a small scikit-learn script and then compare runs in the MLflow UI—all on your laptop, with no cloud setup. It’s designed for class projects, labs, and hackathons where you need a simple, repeatable way to answer: *what did I run and how well did it do?*
```
Comment (qualiaMachine) [inline Learn/Blogs/mlflow-101.qmd:18]: The word "blog" is what should include the hyperlink to https://mlflow.org/docs/latest/tracking.html.

The intro here should talk more about why ML Flow is of value for ML practitioners. Please try to add your own perspective and why you find the tool useful in your experiments. If possible, compare with other similar tools (if you've used them). It doesn't need to be super long -- just long enough that you intrigue people to learn more by reading the linked blog.

File: Learn/Blogs/mlflow-101.qmd [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+#### Prerequisites
+- Intro to Machine Learning with Sklearn (Nexus): https://uw-madison-datascience.github.io/ML-X-Nexus/Learn/Workshops/Intro-ML_Sklearn.html
+- Version Control with GitHub Desktop (Nexus): https://uw-madison-datascience.github.io/ML-X-Nexus/Learn/Guides/Github-desktop.html
+
+Write your blog content here. Discuss the following (if applicable):
+
+- **Motivation**: Many students keep “best results” in scattered notebook cells or screenshots; MLflow makes experiments comparable and reproducible.
```
Comment (qualiaMachine) [inline Learn/Blogs/mlflow-101.qmd:26]: This stuff should be paragraph format, and added to the intro.

**Review by qualiaMachine — COMMENTED** [review]
Hi @Dev-PSR . Thanks so much for your contribution to Nexus! I added a few review comments for you to address before we accept this pull request. Please let me know if anything is unclear in my comments.

File: Learn/Blogs/mlflow-101.qmd [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+  - Code-along
+
+---
+This blog is a beginner-friendly walkthrough of local experiment tracking with [MLflow Tracking](https://mlflow.org/docs/latest/tracking.html). You’ll learn how to log parameters, metrics, and artifacts from a small scikit-learn script and then compare runs in the MLflow UI—all on your laptop, with no cloud setup. It’s designed for class projects, labs, and hackathons where you need a simple, repeatable way to answer: *what did I run and how well did it do?*
+
+#### Prerequisites
+- Intro to Machine Learning with Sklearn (Nexus): https://uw-madison-datascience.github.io/ML-X-Nexus/Learn/Workshops/Intro-ML_Sklearn.html
+- Version Control with GitHub Desktop (Nexus): https://uw-madison-datascience.github.io/ML-X-Nexus/Learn/Guides/Github-desktop.html
+
+Write your blog content here. Discuss the following (if applicable):
```
[the remark this answers is not in the document]
Comment (Dev-PSR) [reply Learn/Blogs/mlflow-101.qmd]: Fixed in commit 9ace32c.

File: Learn/Blogs/mlflow-101.qmd (hunk 3 above)
Comment (Dev-PSR) [reply Learn/Blogs/mlflow-101.qmd]: Fixed in commit 9ace32c.

File: Learn/Blogs/mlflow-101.qmd (hunk 4 above)
Comment (Dev-PSR) [reply Learn/Blogs/mlflow-101.qmd:30]: Fixed in commit 9ace32c.

File: Learn/Blogs/mlflow-101.qmd (hunk 6 above)
Comment (Dev-PSR) [reply Learn/Blogs/mlflow-101.qmd]: Fixed in commit 9ace32c.

File: Learn/Blogs/mlflow-101.qmd (hunk 5 above)
Comment (Dev-PSR) [reply Learn/Blogs/mlflow-101.qmd]: Fixed in commit bc11e94.
